jfrog_platform/README.md

```diff
@@ -0,0 +1,81 @@
+
+
+## Overview
+The following describes how to configure Datadog to gather metrics from JFrog Artifactory and Xray.
+
+### What is JFrog Artifactory and Xray
+JFrog Enterprise with Xray features Artifactory Enterprise and Xray. Together, they empower DevOps teams to improve their productivity to increase velocity and deliver high quality software releases with confidence. Xray scans for known open source security risks and compliance using the industry's most comprehensive intelligence, VulnDB.
```

Comment:
```suggestion

Together, JFrog Artifactory Enterprise and AWS X-Ray empower DevOps teams to improve their productivity to increase velocity and deliver high-quality software releases with confidence. AWS X-Ray scans for known open source security risks and compliance using the industry's most comprehensive intelligence, VulnDB.
```

Reply:
Same as above, I have modified the wording a suggested except for AWS X-Ray